plugin/grpc/grpc.go

```diff
@@ -87,6 +87,11 @@ func (g *GRPC) ServeDNS(ctx context.Context, w dns.ResponseWriter, r *dns.Msg) (
 		return 0, nil
 	}
 
+	if ret == nil {
```

Comment:
this is a good change, and should follow what forward does in forward/forward.go:

~~~
       if upstreamErr != nil {
|       |       return dns.RcodeServerFailure, upstreamErr
|       }           
                    
|       return dns.RcodeServerFailure, ErrNoHealthy
}                   
                 
~~~

Reply:
Replaced everything here to the same error of forward, but instead of creating a new const returned directly a new error.

Reply:
Got it, changed